src/loops/simple_programs_with_repitition/easy/q5/WordCountTest.java

```diff
@@ -0,0 +1,35 @@
+package loops.simple_programs_with_repitition.easy.q5;
+
+import global.variables.*;
+import global.variables.clauses.*;
+import global.BaseTest;
+
+public class WordCountTest extends BaseTest {
+
+    String input = "This is just for testing. Do not copy!";
+
+    public Clause[] testSentence() {
+
+        Clause[] output = new Clause[3];
+        output[0] = new StringLiteral("Enter a sentence: ");
+        output[1] = new NewLine();
+        output[2] = new StringLiteral("The number of words in the sentence is " + countingWords(input) + " words.");
+
+        return output;
+
+    }
+
+    public void runMain() {
+        provideInput(input);
+        WordCount.main(new String[0]);
+    }
+
```

Comment:
Again, you know the number of words you are providing for input. I don't believe you need the actual method.

Reply:
I think this one is fine - eventually, you'll be providing a list of input strings and even if you store an array of their corresponding number of words, you'll still need a method to return the number of words for a specific input string